metadata-editor/app/lib/UsageRightsMetadataMapper.scala

```diff
@@ -0,0 +1,19 @@
+package lib
+
+import com.gu.mediaservice.model._
+
+object UsageRightsMetadataMapper {
+
+  def usageRightsToMetadata(usageRights: UsageRights): Option[ImageMetadata] = {
+    val toImageMetadata: PartialFunction[UsageRights, ImageMetadata] = (ur: UsageRights) => ur match {
+      case u: StaffPhotographer        => ImageMetadata(byline = Some(u.photographer), credit = Some(u.publication))
+      case u: ContractPhotographer     => ImageMetadata(byline = Some(u.photographer), credit = u.publication)
+      case u: CommissionedPhotographer => ImageMetadata(byline = Some(u.photographer), credit = u.publication)
```

Comment:
See `publication` is `String` in StaffPhotographer, but `Option[String]` in `ContractPhotographer`.